src/lib.rs

```diff
@@ -141,7 +151,7 @@ mod tests {
         "#;
 
         let supergraph = Supergraph::new(schema).unwrap();
-        let _subgraphs = database::extract_subgraphs(&supergraph)
+        let _subgraphs = crate::query_graph::extract_subgraphs_from_supergraph::extract_subgraphs_from_supergraph(&supergraph.schema, None)
             .expect("Should have been able to extract subgraphs");
         // TODO: actual assertions on the subgraph once it's actually implemented.
```

Comment:
Is it time to add assertions there? Does JS have corresponding tests?

Reply:
AFAICT this test is not directly in JS, but we could add some that just assert the whole subgraph shape is as expected here.